packages/client/src/views/PrintCertificate/PDFUtils.ts

```diff
@@ -183,12 +183,24 @@ export function executeHandlebarsTemplate(
         return ''
       }
 
-      const id = idParts.join('.')
+      const id =
+        idParts.length === 3
+          ? idParts.slice(0, -1).join('.')
+          : idParts.join('.')
 
-      return intl.formatMessage({
+      const value = intl.formatMessage({
         id,
         defaultMessage: 'Missing translation for ' + id
       })
+
+      if (
```

Comment:
Shouldn't making the result uppercase be made as a subexpression instead? 
https://handlebarsjs.com/guide/expressions.html#subexpressions

```
{{uppercase (intl 'abc')}}
```